apps/examples/testcase/ta_tc/systemio/itc/itc_internal.h

```diff
@@ -37,4 +37,6 @@
 **********************************************************/
 char *Expect_Value(int, int);
 
+int itc_spi_main(void);
```

Comment:
This prototype is for itc, not utc.
So I think this change can be into another commit which you made for itc.

Reply:
The commit to which it is currently added fixes all build breaks. Therefore it is better to keep it here.
The other commit is for renaming an API.